setup.py

```diff
@@ -76,6 +76,20 @@ def run(self):
 
 
 if __name__ == "__main__":
+    extras = {
+        'opt': ['scikit-learn'],
+        'parallel': ['joblib', 'psutil'],
+        'test': ['flake8', 'pytest', 'pytest-cov', ],
+        'docs': ['mne', 'nibabel', 'sphinx', 'nbsphinx', 'sphinx-gallery',
```

Comment:
you should use this in circleci.yml

Reply:
Thanks! Updated the circleci file.

Reply:
I would just add `pooch` and `tqdm` here ... if you want, you can group them with `mne` so it's clear.

Reply:
Sounds good! I'll add them to the docs dependency group.